beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -6633,12 +6635,12 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
             .map_err(Error::InconsistentFork)?;
 
         match fork_name {
-            ForkName::Altair | ForkName::Merge => {
-                LightClientBootstrap::from_beacon_state(&mut state)
+            ForkName::Altair | ForkName::Merge | ForkName::Capella | ForkName::Deneb => {
+                LightClientBootstrap::from_beacon_state(&self.spec, &mut state, &block)
```

Comment:
Could we move `&self.spec` to the last arg here? it's just by convention in Lighthouse - usually `spec` and `logger` are placed as last arguments of a function, for readability